packages/sui-decorators/src/decorators/cache/index.js

```diff
@@ -33,6 +33,13 @@ const _cache = ({
   const cache = caches[fnName]
 
   return (...args) => {
+    if (
+      (typeof window !== 'undefined' && window.__SUI_CACHE_DISABLED__) ||
+      global.__SUI_CACHE_DISABLED__
+    ) {
+      return original
```

Comment:
I think that you should execute the original function.

```
original.apply(instance, args)
```

Reply:
Yep, I quick read the function and I thought that `args` where the decorator `args`. My bad